Automattic/wp-calypso
# Block editor: use WordPress.com brand colors

<img width="1440" alt="Screenshot 2020-06-25 at 22 20 21" src="https://user-images.githubusercontent.com/87168/85830934-17c53900-b796-11ea-88fc-15f79da96e08.png">
(Don't trust the screenshot for exact colors)

#### Changes proposed in this Pull Request

* Adds new FSE plugin that applies wp.com brand colors to block editor



#### Testing instructions

* Spin up sandbox, and from Calypso `apps/full-site-editing` sync files to your sandbox: `yarn dev --sync` (instructions PCYsg-ly5-p2)
* Set feature flag on `define( 'WPCOM_BLOCK_EDITOR_COLORS', true );` in your sandbox file
* Confirm that nothing loads without feature flag
* In block editor, 
  - confirm that primary buttons are pink
  - UI is the correct tone of blue
